front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/Widgets/index.ts

```diff
@@ -87,6 +89,7 @@ export const WIDGET_TITLES: Record<WidgetName, MessageDescriptor> = {
   WhiteSpace: whiteSpaceTitle,
 
   // Homepage builder widgets
+  Areas: areasTitle,
```

Comment:
This doesn't use the areaTerm yet. It requires a little more refactoring and since it's admin-only, I think it's okay.